pkg/utils/k8sutil.go

```diff
@@ -521,6 +523,26 @@ func svcPort(funcObj *kubelessApi.Function) int32 {
 	return int32(8080)
 }
 
+func getPodTemplateFromConfigMap(client kubernetes.Interface) (*v1.PodTemplateSpec, error) {
+	controllerNamespace := os.Getenv("KUBELESS_INSTALLED_NAMESPACE")
```

Comment:
@andresmgot Why do you think so? The idea behind adding here is: since I'm reading a `ConfigMap`(which is a k8s resource), I'm doing it in `k8s-utils.go, where all `k8s` specific operations are done.

Reply:
because these functions are meant to be "kubeless agnostic" (even though some of them may assume things) but with this resolution, this function directly depends on the kubeless installation. I would even move the configuration retrieval to the controller initialization but right now it is fine to leave it here since it's only used in one method now.